Shapes/HelperFunctions.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace Shapes
+{
+    public enum ShapeType
+    {
+        Rhombus,
+        Rectangle,
+        Square,
+        Triangle,
+        Trapezoid
+    };
+    class HelperFunctions
+    {
+        private static void updateFields(Shape shape)
+        {
+            Console.WriteLine("Do you want to fill the shape in the print? y/n");
+            shape.Fill = char.Parse(Console.ReadLine()) == 'y' ? true : false;
+            Console.WriteLine("Do you want to change the char to fill with? y/n (default: #)");
+            char ch = char.Parse(Console.ReadLine());
+            bool toFillWith = ch == 'y' ? true : false;
+            if (toFillWith)
+            {
+                Console.WriteLine("Type the new char you want to change to:");
+                shape.FillWithChar = char.Parse(Console.ReadLine());
+            }
+        }
+        
+        public static Shape CreateShape(ShapeType shapeType) 
+        {
+            FactoryShape factoryShape = new FactoryShape();
+            Shape shape = factoryShape.GetShape(shapeType);
+            switch (shapeType)
+                {
+                case (ShapeType.Rectangle):
+                    Console.WriteLine("Type width:");
+                    ((Rectangle)shape).Width = int.Parse(Console.ReadLine());
+                    Console.WriteLine("Type height:");
+                    ((Rectangle)shape).Height = int.Parse(Console.ReadLine());
```

Comment:
you can create a function in each shape class, that deals with creation of a new shape (with the logic you have here in each case). 

Here you'll call this function for every shape